tests/unit/aiplatform/test_endpoints.py

```diff
@@ -25,6 +25,8 @@
 from google.api_core import operation as ga_operation
```

Comment:
Preference to update system tests with an update call with traffic split. Can be added to the end to end test.